src/axolotl/prompters.py

```diff
@@ -355,6 +355,65 @@ def __init__(
         )
 
 
+CONVERSATION_ROLE_FORMAT = {
+    "chatml": "<|im_start|>{ROLE}",
+    "zephyr": "<|{ROLE}|>",
+}
```

Comment:
If you want to use a role other than chatml, zephyr, add the role format here. This would then use your `from: ___` value into the `{ROLE}` placeholder.